string side
float32 pos_x
float32 pos_y
float32 pos_z
float32 ori_x
float32 ori_y
float32 ori_z
float32 ori_w
